spec/helpers/govuk_link_helper_spec.rb

```diff
@@ -80,11 +80,40 @@ def url_for(path)
       end
     end
 
-    context "when new_tab: true" do
-      let(:kwargs) { { new_tab: true } }
+    describe "new_tab" do
+      let(:expected_new_tab_attributes) { { href: "/world", class: "govuk-link", target: "_blank", rel: "noreferrer noopener" } }
 
-      specify "the new tab attributes are present on the link" do
-        expect(subject).to have_tag("a", text: "hello", with: { href: "/world", class: "govuk-link", target: "_blank", rel: "noreferrer noopener" })
+      context "when new_tab: true" do
```

Comment:
Is there a test for `new_tab: false` too?

Reply:
There is now!